### DISCUSSION TITLE
DACAI display with Thumb Nail display

### DISCUSSION THREAD

**Discussion by I-wish-2** [mriscoc/Ender3V2S1#850] [General]
Greetings,
Using the Configurator.pyw app I compiled this configuration ( Ender3V2-427-BLTUBL-DACAI-IS-LA-MPC-NCDF ).  After flashing the mother board, I inserter an SD card with a simple STL file from Cura, which has the Thumbnail plug-in installed , and when I selected the file to print it did not display the file thumbnail or the QRC how to square.

Is there another step that needs to be completed for this DACAI to display the thumbnails?  Thanks for reading.
D

**Comment by TheWebMachine** [comment]
Did you flash the DACAI display, too? It likely needs a reflash via the microSD slot inside *it's* case. This update will have the preview screen images, such as the qrcode. You are likely missing the necessary screen assets, so the screen isn't being called up.

* Unzip the file below onto a *blank* microSD card...the card cannot have *anything* else on it but these files in the zip: a firmware.bin and a "private" folder with files inside
* Disassemble display and place the card in the slot in back
* Plug display into printer, power on
* Watch for screen to turn blue and update a bunch of files; red text will be sideways (landscape)
* Should end with an "Update Success" message - if error in update process, FULLY FORMAT sd card and copy uncompressed zip again to card
* Turn printer off and disconnect screen
* Remove card from screen and put back together
* Reattach to printer and power on, you should be good to try preview again

If the display doesn't jump into flash mode (blue screen) on boot, you may need to manually repartition the microSD to have a singular partition of only 4GiB FAT if the screen doesn't like the full size/format of your mSD. It didn't like the 64GiB I had handy at default, but worked fine once I repartitioned it to only 4GiB.

https://github.com/mriscoc/Ender3V2S1/blob/Ender3V2S1-Released/display%20assets/dacai_update.zip?raw=true

Hopefully that helps!

**Reply by KobraNeo** [reply]
Is there something missing from the download? I dont see any .bin files in any of the unzipped folders.  I try following your exact instructions and my DACAI wont update at all.

**Reply by mriscoc** [reply]
The file is `firmware.zlib`, you must put this file at root of the SD card.

**Reply by KobraNeo** [reply]
It turns out I dont even have the DACAI. That was the sole source of my errors, and I just have to patiently await support for the TJC.

**Comment by I-wish-2** [comment]
Greetings,
Thank you for directions and the outline to flash the display, well done.
The display issue I thought I had is actually an operator error on my part.  I had previously flashed the motherboard and the DACAI display as an upgrade to my E3 Pro.  At that time the only way to get the display to show the file to be printed in a thumbnail was to upgrade the display to an E3V2 Dwin model.  Now these Dwin displays are as hard to find as hens teeth so I let the thumbnail idea go.

Now with the newest release from Miguel the configurator has the DACAI display as an option when building the configuration to be compiled and I thought that now my old style display could show me the thumbnail of my print file.  Yes it does just fine but my issue was that the Display screen background is black in color and in Cura my Material setting for the PETG filament is also black, Da, Black on Black just did not show the thumbnail.  After some looking at a steep angle to the display I saw a very faint outline.  A color adjustment to the Cura materials color setting resolved the issue perfect.

I'm glad that Miguel received the configuration files from Creality and added them to the current Configurator.pyw.  No more looking for a Dwin display.  Sorry for being wordy and thanks for helping out and reading my I.O. error.
Best Regards
Doug

**Reply by TheWebMachine** [reply]
We've *all* been there! 🤣 Glad you got it figured out. At any rate, this will exist as a quick and dirty FAQ for anyone with a legit need to flash the screen. So, we're still at a net positive. 😎

**Comment by I-wish-2** [comment]
I too have had issues with SD cards over 16 gig.  I was able snag a bunch of 16 gig one so now all is good.  4 and 8 would be better but they too are like the Dwin displays few and far between.  Great information though and thanks.
D

**Reply by TheWebMachine** [reply]
Yeah, I find it's just easier to repartition whatever size I have handy. That also helps to ensure a totally clean format since these low-bit controllers are all too finicky when flashing.

**Reply by I-wish-2** [reply]
Amen.  Nice discussion exchange, that's what makes this site so nice.  Thank you !
Regards
Doug
